wayback-core/src/main/java/org/archive/wayback/archivalurl/ArchivalUrl.java

```diff
@@ -49,19 +49,34 @@ public String toString() {
 		}
 		return toPrefixQueryString(wbRequest.getRequestUrl()); 			
 	}
+    
 	public String toPrefixQueryString(String url) {
 		return toQueryString(url) + STAR;
 	}
+    
 	public String toQueryString(String url) {
 		String datespec = STAR;
-		if((wbRequest.getStartTimestamp() != null) && 
-				(wbRequest.getEndTimestamp() != null)) {
-			datespec = String.format("%s-%s%s",
-					wbRequest.getStartTimestamp(),wbRequest.getEndTimestamp(),
-					STAR);
-		}
-		return toString(datespec,url);
+        
+        String exactDateTimestamp = getEmptyStringIfNull(wbRequest.get(WaybackRequest.REQUEST_EXACT_DATE));
```

Comment:
It is, but as a lot of other places in the code, it is a mixture of tabs and spaces.